mod-orders-storage/examples/piece_collection.sample

```diff
@@ -50,6 +50,7 @@
         "receiptDate": "2018-10-11T00:00:00.000Z",
         "receivingStatus": "Received",
         "supplement": true,
+        "isBound": true
```

Comment:
Choosed isBound. The reason for that from its name whether it is boolean or other type of object will be clear

Second: When Using if statement, I saw many places `if (poLIne.getHolding.. )` from name, it is confusing getHolding will return boolean. `if (poLIne.getIsHolding.. )`  is more clear name for boolean type